policyengine_us/variables/gov/ssa/ssi/eligibility/status/is_ssi_qualified_noncitizen.py

```diff
@@ -0,0 +1,41 @@
+from policyengine_us.model_api import *
+
+
+class is_ssi_qualified_noncitizen(Variable):
+    value_type = bool
+    entity = Person
+    label = "Is an SSI qualified noncitizen"
+    definition_period = YEAR
+    reference = "https://secure.ssa.gov/poms.nsf/lnx/0500502100"
+
+    def formula(person, period, parameters):
+        qualifying_quarters_earnings = person(
+            "ssi_qualifying_quarters_earnings", period
+        )
+        qualifying_quarters_threshold = parameters(
```

Comment:
Then use `p.` for `qualifying_quarters_threshold` and `qualified_statuses`
```suggestion
        p = parameters(period).gov.ssa.ssi
```